### PR TITLE
e2e: save raw profiles too

### PR DESCRIPTION
**Pull request by mikedanese**
#62808

```release-note
NONE
```

### CODE REVIEW

**Comment by wojtek-t** [comment]
@shyamjvs - please review
/assign shyamvs

File: test/e2e/framework/profile_gatherer.go [hunk 1]
```diff
@@ -69,36 +68,44 @@ func gatherProfileOfKind(profileBaseName, kind string) error {
...
 	if err != nil {
-		return fmt.Errorf("Failed to create temp file for profile data: %v", err)
+		return fmt.Errorf("Failed to create file for the profile graph: %v", err)
 	}
-	defer os.Remove(tmpfile.Name())
-	if _, err := tmpfile.Write([]byte(sshResult.Stdout)); err != nil {
+	defer rawprofile.Close()
+
+	if _, err := rawprofile.Write([]byte(sshResult.Stdout)); err != nil {
 		return fmt.Errorf("Failed to write temp file with profile data: %v", err)
```
Comment (shyamjvs) [inline test/e2e/framework/profile_gatherer.go]: s/temp// ?
Here and below.

File: test/e2e/framework/profile_gatherer.go [hunk 2]
```diff
@@ -69,36 +68,44 @@ func gatherProfileOfKind(profileBaseName, kind string) error {
...
-		cmd = exec.Command("go", "tool", "pprof", "-pdf", "-symbolize=none", tmpfile.Name())
-		profilePrefix = "ApiserverCPUProfile_"
+		cmd = exec.Command("go", "tool", "pprof", "-pdf", "-symbolize=none", rawprofile.Name())
 	default:
 		return fmt.Errorf("Unknown profile kind provided: %s", kind)
 	}
 	outfilePath := path.Join(getProfilesDirectoryPath(), profilePrefix+profileBaseName+".pdf")
-	var outfile *os.File
-	outfile, err = os.Create(outfilePath)
+	outfile, err := os.Create(outfilePath)
```
Comment (shyamjvs) [inline test/e2e/framework/profile_gatherer.go]: I think 'err' is already defined above?

Comment (shyamjvs) [reply test/e2e/framework/profile_gatherer.go]: Nvm.. seems like it compiled.

**Comment by shyamjvs** [comment]
nit and lgtm.

**Pull request merged**